tests/installation/bootloader_uefi.pm

```diff
@@ -117,10 +117,8 @@ sub run() {
     }
 
     type_string " \\\n";    # changed the line before typing video params
-                            # https://wiki.archlinux.org/index.php/Kernel_Mode_Setting#Forcing_modes_and_EDID
     bootmenu_default_params;
     specific_bootmenu_params;
-
```

Comment:
also unrelated change. If you don't change it I will merge the PR nevertheless after fixing the typo in the commit message but in general we should keep this in mind so that the history of changes looks clean and we are not changing whitespace back and forth because of personal taste.